Service/Dumper.php

```diff
@@ -89,14 +89,17 @@ public function dump($targetDir, $host, $section = null, array $options = array(
         }
 
         foreach ($this->urlsets as $urlset) {
-            $urlset->save($this->tmpFolder, $options['gzip']);
+            if ($urlset instanceof DumpingUrlset) {
```

Comment:
Should we throw an exception if `Dumper` got something else than `DumpingUrlset`?

Reply:
We may, but this is not the purpose of this PR.